backend/__fixtures__/configmaps.js

```diff
@@ -0,0 +1,110 @@
+//
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+const { cloneDeep, find, filter, split, isEmpty } = require('lodash')
+const createError = require('http-errors')
+const pathToRegexp = require('path-to-regexp')
+const { createUrl } = require('./helper')
+
+function getConfigMap ({ namespace, name, labels, creationTimestamp, data = {} }) {
+  const metadata = {
+    namespace,
+    name
+  }
+  if (!isEmpty(labels)) {
+    metadata.labels = labels
+  }
+
+  if (creationTimestamp) {
+    metadata.creationTimestamp = creationTimestamp
+  }
+
+  return { metadata, data }
+}
+
+function getClusterIdentitConfigMap ({ identity = 'landscape-test', ...options } = {}) {
+  return getConfigMap({
+    namespace: 'kube-system',
+    name: 'cluster-identity',
+    data: {
+      'cluster-identity': identity
+    },
+    ...options
+  })
+}
+
+const configMapsList = [
+  getClusterIdentitConfigMap()
+]
+
+const configMaps = {
+  get (namespace, name) {
+    const items = configMaps.list(namespace)
+    return find(items, ['metadata.name', name])
+  },
+  list (namespace) {
+    const items = cloneDeep(configMapsList)
+    return namespace
+      ? filter(items, ['metadata.namespace', namespace])
+      : items
+  },
+  getClusterIdentityConfigMap (identity) {
+    return getClusterIdentitConfigMap({ identity })
+  }
```

Comment:
```suggestion
  createClusterIdentityConfigMap (identity) {
    return getClusterIdentitConfigMap({ identity })
  }
```
The convention for other resources is to name this `create`. In this case I would propose to rename to `createClusterIdentityConfigMap`

Reply:
✅